lib/spack/external/spack_astunparse/unparser.py

```diff
@@ -74,14 +84,22 @@ def write(self, text):
         "Append a piece of text to the current line."
         self.f.write(six.text_type(text))
 
-    def enter(self):
-        "Print ':', and increase the indentation."
-        self.write(":")
-        self._indent += 1
+    class _Block:
+        """A context manager for preparing the source for blocks. It adds
+        the character':', increases the indentation on enter and decreases
```

Comment:
(minor) this is a typo (no space between "character" and "':'". More importantly, I'm curious if this comment will be displayed in the PR (I made this comment on 493c867)

Reply:
I can read it when looking at the PR "conversation" tab, so I believe the answer is yes!